karaf/Dockerfile.rhel

```diff
@@ -27,7 +27,6 @@ USER root
 ADD jboss.repo /etc/yum.repos.d/jboss.repo
 RUN yum install -y --disablerepo=\* --enablerepo=jboss maven \
  && rpm -e --nodeps java-1.7.0-openjdk-devel java-1.7.0-openjdk java-1.7.0-openjdk-headless \
```

Comment:
Java 7 is used only by Maven since it is introduced by it. Of course, since Maven 3.0.5 has been productised with Java 7, this probably slightly break this because Maven is now used with Java 8 (the image's JDK). But that is the same net effect as when setting Java 8 as the default JVM.

See also https://issues.jboss.org/browse/CLOUD-46 for a discussion about it.

Unfortunately this remove doesn't remove all these transitive dependencies of Java 7 (X11,GTK, cups, perl, ...), too, so the image is still quite large.